ops/gulp/pkg-css.js

```diff
@@ -126,24 +100,21 @@ var minifyTask = (cb) => {
         .pipe(rename({
             suffix: `.min`
         }))
-        .pipe(gulp.dest(paths.dest))
+        .pipe(gulp.dest(paths.dest));
+    cb();
 }
 gulp.task('pkg-css-minify', minifyTask);
 
 //add banner
-var bannerTask = (cb) => {
-    return gulp.src([`${paths.dest}/**/*.css`])
+const bannerTask = (cb) => {
+    gulp.src([`${paths.dest}/**/*.css`])
         .pipe(header(banner))
-        .pipe(gulp.dest(paths.dest))
-}
+        .pipe(gulp.dest(paths.dest));
+    cb();
+};
 gulp.task('pkg-css-banner', bannerTask);
 
 //main css task
-module.exports = cssTask = (cb) => {
-    if (environment.production) {
-        gulpSequence('pkg-sass', 'pkg-dark-sass', 'pkg-css-components', 'pkg-css-minify', 'pkg-css-banner', cb)
-    } else {
-        gulpSequence('pkg-sass', 'pkg-dark-sass', cb)
-    }
-}
-gulp.task('pkg-css', cssTask);
+module.exports = gulp.task('pkg-css', environment.production ?
+    gulp.series('pkg-sass', 'pkg-dark-sass', 'pkg-css-components', 'pkg-css-minify', 'pkg-css-banner') :
+    gulp.series('pkg-sass'));
```

Comment:
Because of https://github.com/SAP/fundamental/pull/948/files#r231978094 I had to remove ```pkg-dark-sass``` from here. @saadmhybris can you please help here?